ppr-ui/src/resources/tableHeaders.ts

```diff
@@ -260,3 +260,36 @@ export const partyTableHeaders: Array<BaseHeaderIF> = [
     value: 'edit'
   }
 ]
+
+export const registeringTableHeaders: Array<BaseHeaderIF> = [
+  {
+    class: 'column-mds',
+    sortable: false,
+    text: 'Name',
+    value: 'name'
+  },
+  {
+    class: 'column-md',
+    sortable: false,
+    text: 'Address',
+    value: 'address'
+  },
+  {
+    class: 'column-md',
+    sortable: false,
+    text: 'Email Address',
+    value: 'emailAddress'
+  },
+  {
+    class: 'column-mds',
+    sortable: false,
+    text: 'Registering Party Code',
+    value: 'code'
+  },
+  {
```

Comment:
Probably don't need this edit column

Reply:
Leaving for now: ui may change to allow edits on RP.